schema.ts

```diff
@@ -1088,6 +1088,19 @@ export type ContextProperties = Array<PropertyIdentifier<string>>;
 
 export type BasicIndexedProperty = PropertyIdentifier<string>;
 
+/**
+ * A property that will be used to filter the results of a search.
+ * 
+ * For example, if you are trying to query for all emails last modified in the last 30 days,
+ * you would set the filterableProperties to the property that represents the last modified date.
+ * 
+ * Filterable properties must be one of type 
+ * {@link ValueHintType.SelectList} {@link ValueType.Boolean} {@link ValueHintType.DateTime} or 
+ * {@link ValueType.Number} (Floats will be supported in the future, today they will be truncated to an integer)
```

Comment:
```suggestion
 * Filterable properties must be one of the following types: 
 * {@link ValueHintType.SelectList}, {@link ValueType.Boolean}, {@link ValueHintType.DateTime} or 
 * {@link ValueType.Number}. For filtering purposes, number values will be rounded down to the nearest integer.
```

Do we know if it's round or round down?

Reply:
I assume since we're going to clamp it will be a round down.

Reply:
It's up to us, it's whatever logic we use when we load into TP, so we can do whatever your preference is. Round down sounds reasonable to me (keeping only the integer part as our casting logic).